Formula/d/dadroit_json_generator.rb

```diff
@@ -0,0 +1,41 @@
+class DadroitJsonGenerator < Formula
+  desc "High-performance tool for generating nested JSON data using custom templates"
+  homepage "https://github.com/DadroitOrganization/Generator"
+  url "https://github.com/DadroitOrganization/Generator/archive/refs/tags/Release_Version_1.0.0.352.tar.gz"
+  sha256 "2ef4e55bb00d5a61dad40bc050de7b8d56611e777f9db10c3e9997d5a2ade156"
+  license "GPL-3.0-or-later"
```

Comment:
When provided under the open-source option, is this software released under exactly the terms of the GPL? From [your license text](https://github.com/DadroitOrganization/Generator/blob/30c395e2973c3d23cdfd8e0dffb92eefafa15620/LICENSE.txt), I see:

> Under “GPL v3 or later” with some additional changes

I take it that the following is meant to be _in addition_ to the GPL:

> You are granted a non-exclusive, non-transferable, revocable license to access and use our service strictly in accordance with these terms of conditions. As a condition of your use of the service, you warrant to Dadroit that you will not use the service for any purpose that is unlawful or prohibited by this document. You may not use the site in any manner which could damage, disable, overburden, or impair the usage or interfere with any other party's use of the service. You must not obtain or attempt to obtain any materials or information through any means not intentionally made available or provided for through the service. You will not modify, publish, transmit, reverse engineer, participate in the transfer or sale, create derivative works, or in any way exploit any of the content, in whole or in part, found on the service. Dadroit website or applications cannot be used for resale. Your use of the service does not entitle you to make any unauthorized use of any protected content, and in particular, you will not delete or alter any proprietary rights or attribution notices in any content. You will use protected content solely for your personal use and will make no other use of the content without the express written permission of Dadroit and other copyright owners. You agree that you do not acquire any ownership rights in any protected content. We do not grant you any licenses, expressed or implied, to the intellectual property of Dadroit except as explicitly authorized by Dadroit.

If so, this should be probably be mentioned/expressed here (and maintainers should take a look).